### PR TITLE
gcp-observability: add new *compressed_bytes_per_rpc views

### PR DESCRIPTION
**Pull request by DNVindhya**
This PR adds new views for payload metrics and registers new views along with latency related views for gcp-observability. 

List of new views for payload metrics:
Client Views
* `grpc.io/client/sent_compressed_message_bytes_per_rpc`
* `grpc.io/client/received_compressed_message_bytes_per_rpc`

Server Views
* `grpc.io/server/sent_compressed_message_bytes_per_rpc`
* `grpc.io/server/received_compressed_message_bytes_per_rpc`

Note: Payload related views are defined in `grpc-census` temporarily. This will be moved to [`opencensus-java`](https://github.com/census-instrumentation/opencensus-java) repository later.

The above listed views along with latency related views are enabled for `gcp-observability`.

CC @sanjaypujare @ejona86

### CODE REVIEW

File: census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+              1073741824.0,
+              4294967296.0));
+
+  static final Aggregation AGGREGATION_WITH_BYTES_HISTOGRAM =
+      Distribution.create(BucketBoundaries.create(RPC_BYTES_BUCKET_BOUNDARIES));
+
+  public static final View GRPC_CLIENT_SENT_COMPRESSED_MESSAGE_BYTES_PER_RPC =
+      View.create(
+          View.Name.create("grpc.io/client/sent_compressed_message_bytes_per_rpc"),
+          "Sent compressed message bytes per RPC",
```
Comment (sanjaypujare) [inline census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java:64]: Can you include "client" or "client side" in the description? Also for line 72.

And "server side" for lines 80 and 88

Comment (DNVindhya) [reply census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java]: Updated description to "Compressed message bytes sent per client RPC attempt" and similarly for other client and server views.

File: census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              262144.0,
+              1048576.0,
+              4194304.0,
+              16777216.0,
+              67108864.0,
+              268435456.0,
+              1073741824.0,
+              4294967296.0));
+
+  static final Aggregation AGGREGATION_WITH_BYTES_HISTOGRAM =
```
Comment (ejona86) [inline census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java:58]: FWIW, I think you could do `RpcViewConstants.GRPC_CLIENT_SENT_BYTES_PER_RPC_VIEW.getAggregation()` to get this. Not too important.

Comment (DNVindhya) [reply census/src/main/java/io/grpc/census/internal/ObservabilityCensusConstants.java:37]: Done

**Review by sanjaypujare — APPROVED** [review]
LGTM

**Pull request merged**